### ISSUE TITLE
Idea: garbage collection times

### ISSUE THREAD

**Issue by david4worx** [PropaneDragon/RushHour#10]
Right now garbage collection is done quite randomly. At least, it feels that way. With the mod enabled and having garbage pickup like whenever things can get quite messy with the traffic, especially during rush hour.
Proposed solution is to start garbage collection late at night, early mornings (think between 01:00 or 04:00) or maybe even better, make it configurable.

**Comment by PropaneDragon** [comment]
Garbage collection is done on an as-need basis like the rest of the simulation in Cities. I don't think I'll be able to change this without a lot of work unfortunately, but I'll look into it in the future. Thanks

**Comment by david4worx** [comment]
Thought it would. Feels like garbage, death and illness are a separate subject and not really compare in code as education for example. And this is from the very very limited browsing I've done in the codebase.

**Comment by david4worx** [comment]
@PropaneDragon brainfart, wouldn't it be possible to just return early (false) on the dispatching of garbage trucks outside of a given timeframe? It should be able to re-use parts of the code responsible for not dispatching any more garbage trucks when the limit is reached, right? Just came to this as I went over parts of the ARIS overwatch mod, which hooks into the same parts.

**Comment by PropaneDragon** [comment]
This is possible, but wouldn't trash become a huge issue at about mid day? Granted it would be cleared up at whatever time the garbage trucks set off, but it'll still be a bit of a mess.

**Comment by david4worx** [comment]
I don't think it will be any different compared to the industry/commercial goods requirements.
Might be worth a try right?
If it does become a problem, there is always an option to reduce the garbage modifier for buildings, correct?

**Comment by PropaneDragon** [comment]
Yeah, good point.
